app/src/main/java/com/jerboa/ui/components/post/PostActivity.kt

```diff
@@ -295,6 +298,17 @@ fun PostActivity(
                                     .testTag("jerboa:comments"),
                         ) {
                             item(key = "${postView.post.id}_listing", "post_listing") {
+                                val instantScores =
```

Comment:
What's the reason these are getting moved around?

Reply:
> What's the reason these are getting moved around?

Honestly that wasn't my best approach, agree to that. Would be better to pass VoteType to child composables